packages/devextreme/js/__internal/ui/html_editor/ui/m_formDialog.ts

```diff
@@ -1,21 +1,13 @@
 import localizationMessage from '@js/common/core/localization/message';
-import devices from '@js/core/devices';
+import type { dxElementWrapper } from '@js/core/renderer';
 import $ from '@js/core/renderer';
-import type { DeferredObj } from '@js/core/utils/deferred';
-import { Deferred } from '@js/core/utils/deferred';
 import { extend } from '@js/core/utils/extend';
-import {
-  // @ts-expect-error
-  getCurrentScreenFactor,
-  hasWindow,
-} from '@js/core/utils/window';
 import Form from '@js/ui/form';
-import Popup from '@js/ui/popup';
 import { isFluent, isMaterialBased } from '@js/ui/themes';
 
-const DIALOG_CLASS = 'dx-formdialog';
+import DialogBase from './m_baseDialog';
```

Comment:
When defining a FormDialog, you use a modifier before the root 'dialog'. Please rename this class consistently.

```suggestion
import BaseDialog from './m_baseDialog';
```